tidb-lightning/tidb-lightning-log-message-explanation.md

```diff
@@ -0,0 +1,360 @@
+---
+title: TiDB Lightning Log Message Explanation
+summary: Provide detailed explanation of log messages generated during the importing process using TiDB Lightning.
+---
+
+# TiDB Lightning Log Message Explanation
+
+Based on a successful test data import, this document explains log messages of **TiDB Lightning v5.4** with **Local Backend mode**, and dives deep to understand where the log comes from and what it actually represents. You can refer to this doc to better understand TiDB Lightning logs.
+
+To read this document, you are expected to be already familiar with TiDB Lightning and have prior knowledge of its high-level workflow described in [TiDB Lightning Overview](/tidb-lightning/tidb-lightning-overview.md). If you encounter unfamiliar terminology, you can refer to the [glossary](/tidb-lightning/tidb-lightning-glossary.md).
+
+You could use this doc to quickly navigate within TiDB Lightning source code and better understand how it works internally and what exactly the log message means.
+
+Note that some trivial logs are ignored. Only important logs are included in the following doc.
+
+## Log message explanation 
+
+```
+[INFO] [info.go:49] ["Welcome to TiDB-Lightning"] [release-version=v5.4.0] [git-hash=55f3b24c1c9f506bd652ef1d162283541e428872] [git-branch=HEAD] [go-version=go1.16.6] [utc-build-time="2022-04-21 02:07:55"] [race-enabled=false]
+```
+
+[info.go:49](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/version/build/info.go#L49): Print TiDB Lightning version information.
```

Comment:
Can you provide permanent links here? I was told these packages would soon be moved to somewhere else. I'm concerned these links will be broken very soon. [ref](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/version/build/info.go#L49)